src/platform/site-wide/user-nav/components/HelpMenu.jsx

```diff
@@ -8,7 +8,7 @@ import isBrandConsolidationEnabled from '../../../brand-consolidation/feature-fl
 class HelpMenu extends React.Component {
   render() {
     const buttonText = isBrandConsolidationEnabled() ? 'Contact Us' : 'Help';
-    const icon = <IconHelp color="#fff"/>;
+    const icon = <IconHelp color="#fff" role="presentation"/>;
```

Comment:
Should the `role` property be part of the `IconBase` class?

Reply:
Are icons always presented with accompanying text? I think it's safe to add the default `role="presentation"` to the `IconBase` if so. We can still override it for links and whatnot. I'm not sure when we'd want for the icon to be read out by a screen reader by default.

Reply:
@cvalarida Icons aren't always presented with text, no. The Search icon when expanded is a good example. The icon is the only thing present, so having it be role="presentation" would be wrong. Which means on further review, my code is also wrong on this PR in one instance. I'll fix it and comment after lunch, so I can still make a push to master this afternoon yet.